.github/workflows/release.yml

```diff
@@ -51,6 +51,15 @@ jobs:
             echo "Tag should match pom.xml project.version"
             exit 1
           fi
+      - uses: ./.github/workflows/maven-goal
+        with:
+          command: './mvnw -q help:evaluate -Dexpression=project.version -DforceStdout'
+      - name: Validate version is a release version
+        run: |
+          if [[ "$(./mvnw -q help:evaluate -Dexpression=project.version -DforceStdout)" =~ "-SNAPSHOT" ]]; then
```

Comment:
This command outputs the following:
```
Apache Maven 3.8.5 (3599d3414f046de2324203b78ddcf9b5e4388aa0)
Maven home: /home/sylvain/.m2/wrapper/dists/apache-maven-3.8.5-bin/5i5jha092a3i37g0paqnfr15e0/apache-maven-3.8.5
Java version: 17.0.8, vendor: Eclipse Adoptium, runtime: /home/sylvain/.sdkman/candidates/java/17.0.8-tem
Default locale: en_US, platform encoding: UTF-8
OS name: "linux", version: "6.5.0-17-generic", arch: "amd64", family: "unix"
1.47.1-SNAPSHOT%
```

Thus only the last line is relevant here, maybe it's worth adding an `| tail -1` to only get the last line to ensure only the version is checked here, even if it's a bit unlikely that the `SNAPSHOT` string could end up here unless a snapshot maven version is used.

Reply:
I cannot get the same outcome locally:

```bash
$ git --no-pager log -n 1
commit b3d7c6d20ce51e900daee5fb95b4a8a035ff3270 (HEAD -> main, upstream/main, origin/main)

$ ./mvnw -q help:evaluate -Dexpression=project.version -DforceStdout 2> /dev/null
1.6.1-SNAPSHOT

$ ./mvnw -q help:evaluate -Dexpression=project.version -DforceStdout 1> /dev/null
The operation couldn’t be completed. Unable to locate a Java Runtime.
Please visit http://www.java.com for information on installing Java.
```

Reply:
can you confirm whether the stdout contains only the version?

Reply:
I forgot that this depends on the local maven config here https://github.com/elastic/apm-pipeline-library/issues/2186 was the similar issue where we had to deal with the same kind of output. do we have a per-repository maven config that is used for maven options in CI ?

Reply:
I found where it came from: I was testing on another repository where the maven config actually has the `-V` which makes the output explicitly verbose here, that's something we have to remind removing when opening the PR on the APM agent.

https://github.com/elastic/apm-agent-java/blob/main/.mvn/maven.config